database/src/main/resources/db/migration/V6_0__add_marker_tfr_constraint.sql

```diff
@@ -0,0 +1,2 @@
+CREATE UNIQUE INDEX IF NOT EXISTS marker_transfer_txhash_idx ON marker_transfer(tx_hash);
+ALTER TABLE marker_transfer ADD CONSTRAINT unique_tfr_txhash UNIQUE USING INDEX marker_transfer_txhash_idx;
```

Comment:
would you mind explaining how this is different than line 1?

Reply:
The conflict resolution code requires a unique constraint, not a unique index. The unique constraint has to be added on a unique index.